### DISCUSSION TITLE
Question: encoding problem when GenerateJSONReport function is used

### DISCUSSION THREAD

**Discussion by Conanien** [Qualys/log4jscanwin#33] [Q&A]
I would have question regarding GenerateJSONReport function. You are using RapidJson module for your project where we observed that many of filepaths in json reports generated by scanner are not showing special characters properly.

Lets take this string as an example: APPLICAÇÕES

When I use RapidJson module alone and I feed string buffer directly, results are shown as expected.

 But when whole scanner is used and this string is in filepath, report will provide different set of characters instead of Ç and Õ.

Is this intended?

**Comment by romw** [comment]
That is not the intended behavior.  We'll need to investigate.
